Sprint-3/3-mandatory-practice/implement/repeat.js

```
@@ -1,5 +1,7 @@
-function repeat() {
-    return "hellohellohello";
+function repeat(str, count) {
+  if (count === 0) return "";
+  else if (count < 0) return "Please enter positive number!";
+  else return str.repeat(count);
```

How would the caller distinguish the following two function calls?
1. `repeat("Please enter positive number!", 1)`
2. `repeat("", -1)`

Both function calls return the same value.

---

We can throw new Error(`Please enter positive number`), instead of return string`Please enter positive number!`